src/rgw/rgw_op.cc

```diff
@@ -7266,7 +7277,9 @@ void RGWDeleteMultiObj::handle_individual_object(const rgw_obj_key& o, optional_
   if (op_ret == -ENOENT) {
     op_ret = 0;
   }
-
+  if (op_ret < 0) {
+    return;
+  }
```

Comment:
`RGWDeleteMultiObj::handle_individual_object()` needs to communicate this error to `send_partial_response()`